pandas/core/frame.py

```diff
@@ -2552,7 +2552,8 @@ def to_stata(
     @deprecate_kwarg(old_arg_name="fname", new_arg_name="path")
     def to_feather(self, path: FilePathOrBuffer[AnyStr], **kwargs) -> None:
         """
-        Write a DataFrame to the binary Feather format.
+        Write a DataFrame to the binary Feather format. Requires a default index.
```

Comment:
these changes should be in Notes. this should have failed the doc-string checker (as this is no longer a summary line). are we still running these checks, cc @MarcoGorelli

Reply:
@jreback can you please clarify what do you mean with Notes? I searched the repo for the docs and this is the only place I've found so I updated it. Do you perhaps mean that you want a summary line for the first sentence followed by an empty line with more details?

Reply:
Notes section, you have to add it. yes summary is 1 line.

Reply:
OK, added what I understand you mean by "notes section" - another paragraph. Let me know if there's something else I can do.